samples/jni/Application.mk

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+APP_ABI := armeabi-v7a arm64-v8a x86 x86_64
```

Comment:
My one question is how long does the kokoro build take with these 4, if it's still relatively fast (ie no slower then the windows builds) then this is fine, otherwise we may want to cut the list down.

Reply:
550 sec windows build versus 930 sec for kokoro NDK on all ABIs, so it is worth limiting to a single ABI.

I picked up arm64-v8a for both android_test and android_ndk, given that this is the ABI used of most Android devices offering Vulkan support (the Nvidia shield is the only exception I know).